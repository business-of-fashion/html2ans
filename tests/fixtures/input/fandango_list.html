<body>
<p>Some movies to see!</p>
<p>
<ul>
    <li><a href="http://www.qksrv.net/click-7998314-10576771?url=http%3a%2f%2fwww.fandango.com%2fcarol_184244%2fmovietimes%3flocation%3d97205%26wssaffid%3d11836%26wssac%3d123">Carol</a></li>
    <li><a href="http://www.qksrv.net/click-7998314-10576771?url=http%3a%2f%2fwww.fandango.com%2fthe%2bdanish%2bgirl_136694%2fmovietimes%3flocation%3d97205%26wssaffid%3d11836%26wssac%3d123">The Danish Girl</a></li>
    <li><a href="http://www.qksrv.net/click-7998314-10576771?url=http%3a%2f%2fwww.fandango.com%2fhow%2bto%2bbe%2bsingle_185754%2fmovietimes%3flocation%3d97205%26wssaffid%3d11836%26wssac%3d123">How to be Single</a></li>
    <li><a href="http://www.qksrv.net/click-7998314-10576771?url=http%3a%2f%2fwww.fandango.com%2fthe%2blady%2bin%2bthe%2bvan_181204%2fmovietimes%3flocation%3d97205%26wssaffid%3d11836%26wssac%3d123">The Lady in the Van</a></li>
    <li><a href="http://www.qksrv.net/click-7998314-10576771?url=http%3a%2f%2fwww.fandango.com%2fthe%2brevenant_179969%2fmovietimes%3flocation%3d97205%26wssaffid%3d11836%26wssac%3d123">The Revenant</a></li>
    <li><a href="http://www.qksrv.net/click-7998314-10576771?url=http%3a%2f%2fwww.fandango.com%2froom%2b_185579%2fmovietimes%3flocation%3d97205%26wssaffid%3d11836%26wssac%3d123">Room </a></li>
    <li><a href="http://www.qksrv.net/click-7998314-10576771?url=http%3a%2f%2fwww.fandango.com%2fspotlight_184742%2fmovietimes%3flocation%3d97205%26wssaffid%3d11836%26wssac%3d123">Spotlight</a></li>
    <li><a href="http://www.qksrv.net/click-7998314-10576771?url=http%3a%2f%2fwww.fandango.com%2ftouched%2bwith%2bfire_187919%2fmovietimes%3flocation%3d97205%26wssaffid%3d11836%26wssac%3d123">Touched With Fire</a></li>
    <li><a href="http://www.qksrv.net/click-7998314-10576771?url=http%3a%2f%2fwww.fandango.com%2fwhere%2bto%2binvade%2bnext_188065%2fmovietimes%3flocation%3d97205%26wssaffid%3d11836%26wssac%3d123">Where to Invade Next</a></li>
    <li><a href="http://www.qksrv.net/click-7998314-10576771?url=http%3a%2f%2fwww.fandango.com%2fthe%2bwitch_185794%2fmovietimes%3flocation%3d97205%26wssaffid%3d11836%26wssac%3d123">The Witch</a></li>
    <li><a href="http://www.qksrv.net/click-7998314-10576771?url=http%3a%2f%2fwww.fandango.com%2fzoolander%2b2_181503%2fmovietimes%3flocation%3d97205%26wssaffid%3d11836%26wssac%3d123">Zoolander 2</a></li>
</ul>
</p>
<p><a href="http://www.qksrv.net/click-7998314-10576771?url=http%3a%2f%2fwww.fandango.com%2fmovietimes%3fwssaffid%3d11836%26wssac%3d123">Search other theaters</a></p>
</body>