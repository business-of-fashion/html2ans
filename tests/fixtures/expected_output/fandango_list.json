[
  {
    "type": "text",
    "content": "Some movies to see!"
  },
  {
    "type": "list",
    "list_type": "unordered",
    "items": [
      {
        "type": "text",
        "content": "Carol",
        "additional_properties": {
          "href": "http://www.qksrv.net/click-7998314-10576771?url=http%3a%2f%2fwww.fandango.com%2fcarol_184244%2fmovietimes%3flocation%3d97205%26wssaffid%3d11836%26wssac%3d123"
        }
      },
      {
        "type": "text",
        "content": "The Danish Girl",
        "additional_properties": {
          "href": "http://www.qksrv.net/click-7998314-10576771?url=http%3a%2f%2fwww.fandango.com%2fthe%2bdanish%2bgirl_136694%2fmovietimes%3flocation%3d97205%26wssaffid%3d11836%26wssac%3d123"
        }
      },
      {
        "type": "text",
        "content": "How to be Single",
        "additional_properties": {
          "href": "http://www.qksrv.net/click-7998314-10576771?url=http%3a%2f%2fwww.fandango.com%2fhow%2bto%2bbe%2bsingle_185754%2fmovietimes%3flocation%3d97205%26wssaffid%3d11836%26wssac%3d123"
        }
      },
      {
        "type": "text",
        "content": "The Lady in the Van",
        "additional_properties": {
          "href": "http://www.qksrv.net/click-7998314-10576771?url=http%3a%2f%2fwww.fandango.com%2fthe%2blady%2bin%2bthe%2bvan_181204%2fmovietimes%3flocation%3d97205%26wssaffid%3d11836%26wssac%3d123"
        }
      },
      {
        "type": "text",
        "content": "The Revenant",
        "additional_properties": {
          "href": "http://www.qksrv.net/click-7998314-10576771?url=http%3a%2f%2fwww.fandango.com%2fthe%2brevenant_179969%2fmovietimes%3flocation%3d97205%26wssaffid%3d11836%26wssac%3d123"
        }
      },
      {
        "type": "text",
        "content": "Room",
        "additional_properties": {
          "href": "http://www.qksrv.net/click-7998314-10576771?url=http%3a%2f%2fwww.fandango.com%2froom%2b_185579%2fmovietimes%3flocation%3d97205%26wssaffid%3d11836%26wssac%3d123"
        }
      },
      {
        "type": "text",
        "content": "Spotlight",
        "additional_properties": {
          "href": "http://www.qksrv.net/click-7998314-10576771?url=http%3a%2f%2fwww.fandango.com%2fspotlight_184742%2fmovietimes%3flocation%3d97205%26wssaffid%3d11836%26wssac%3d123"
        }
      },
      {
        "type": "text",
        "content": "Touched With Fire",
        "additional_properties": {
          "href": "http://www.qksrv.net/click-7998314-10576771?url=http%3a%2f%2fwww.fandango.com%2ftouched%2bwith%2bfire_187919%2fmovietimes%3flocation%3d97205%26wssaffid%3d11836%26wssac%3d123"
        }
      },
      {
        "type": "text",
        "content": "Where to Invade Next",
        "additional_properties": {
          "href": "http://www.qksrv.net/click-7998314-10576771?url=http%3a%2f%2fwww.fandango.com%2fwhere%2bto%2binvade%2bnext_188065%2fmovietimes%3flocation%3d97205%26wssaffid%3d11836%26wssac%3d123"
        }
      },
      {
        "type": "text",
        "content": "The Witch",
        "additional_properties": {
          "href": "http://www.qksrv.net/click-7998314-10576771?url=http%3a%2f%2fwww.fandango.com%2fthe%2bwitch_185794%2fmovietimes%3flocation%3d97205%26wssaffid%3d11836%26wssac%3d123"
        }
      },
      {
        "type": "text",
        "content": "Zoolander 2",
        "additional_properties": {
          "href": "http://www.qksrv.net/click-7998314-10576771?url=http%3a%2f%2fwww.fandango.com%2fzoolander%2b2_181503%2fmovietimes%3flocation%3d97205%26wssaffid%3d11836%26wssac%3d123"
        }
      }
    ]
  },
  {
    "type": "interstitial_link",
    "content": "Search other theaters",
    "additional_properties": {
      "href": "http://www.qksrv.net/click-7998314-10576771?url=http%3a%2f%2fwww.fandango.com%2fmovietimes%3fwssaffid%3d11836%26wssac%3d123"
    },
    "url": "http://www.qksrv.net/click-7998314-10576771?url=http%3a%2f%2fwww.fandango.com%2fmovietimes%3fwssaffid%3d11836%26wssac%3d123"
  }
]